epam-promo/components/inputs/TextArea.tsx

```diff
@@ -1,29 +1,19 @@
 import * as React from 'react';
 import { withMods } from '@epam/uui-core';
-import { TextArea as uuiTextArea, TextAreaProps } from '@epam/uui-components';
-import * as types from '../types';
-import css from './TextArea.scss';
+import { TextArea as uuiTextArea, TextAreaProps as UuiTextAreaProps, TextAreaMods as UuiTextAreaMods } from '@epam/uui';
 
-const defaultSize = '36';
-const defaultMode = types.EditMode.FORM;
-
-export interface TextAreaMods extends types.IHasEditMode {
-    size?: types.ControlSize;
+export interface TextAreaMods {
```

Comment:
These mods already exist in UuiTextAreaProps, so you don't need to duplicate in here.

Reply:
fixed